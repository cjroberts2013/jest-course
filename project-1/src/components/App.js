import React, { Component } from 'react'

class App extends Component {
    render() {
        return (
            <div>
                <h2>Gift Giver App</h2>
            </div>
        )
    }
}

export default App;