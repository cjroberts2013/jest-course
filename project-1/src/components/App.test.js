import React from 'react';
import { shallow, mount, render } from 'enzyme';
import toJson from 'enzyme-to-json';
import App from './App';




const app = shallow(<App />);

it('renders correctly', () => {
    expect(toJson(app)).toMatchSnapshot();
})

