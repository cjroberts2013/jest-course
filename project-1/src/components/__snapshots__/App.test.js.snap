// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h2>
    Gift Giver App
  </h2>
</div>
`;
